### PR TITLE
feat: hpmn networks

### PR DESCRIPTION
**Pull request by strophy**
## Issue being fixed or feature implemented

Dash Core 19.0.0-beta.5 enables support for HPMN nodes

## What was done?

Implement HPMN

## How Has This Been Tested?

On devnets

## Breaking Changes

Changes to `generate` command syntax

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: ansible/deploy.yml [hunk 1]
```diff
@@ -171,16 +195,39 @@
...
+# Register masternodes and set sporks
+
 - name: Register masternodes
   hosts: wallet_nodes
   become: true
   roles:
-    - mn_init
+    - role: mn_init
+      mn_group_name: masternodes
+      # mnlist: {{ hp_masternodes }}
```
Comment (shumkov) [inline ansible/deploy.yml:206]: Not sure what does it mean?

File: ansible/group_vars/all [hunk 2]
```diff
@@ -53,7 +53,7 @@ tenderdash_log_level: debug
 dapi_image: dashpay/dapi
 dapi_envoy_image: envoyproxy/envoy
 dapi_port: 3000
-dapi_grpc_port: 3010
+# dapi_grpc_port: 3010
```
Comment (shumkov) [inline ansible/group_vars/all:56]: You can just remove it.

File: ansible/deploy.yml (hunk 1 above)
Comment (strophy) [reply ansible/deploy.yml:206]: I need help with this bit

File: ansible/group_vars/all (hunk 2 above)
Comment (strophy) [reply ansible/group_vars/all:56]: Update to what value?

File: package.json [hunk 3]
```diff
@@ -21,11 +21,11 @@
   },
   "homepage": "https://github.com/dashpay/dash-network-deploy#readme",
   "dependencies": {
-    "@dashevo/dapi-client": "^0.23.0",
+    "@dashevo/dapi-client": "^0.23.2",
```
[the remark this answers is not in the document]
Comment (strophy) [reply package.json:24]: I have a separate `chore/update-deps` branch to bump all deps ready and tested, you think we should include it in this PR?

[the remark this answers is not in the document]
Comment (shumkov) [reply package.json:24]: Not necessary

**Review by shumkov — APPROVED** [review]

**Pull request merged**